content/docs/gke/authentication.md

```diff
@@ -0,0 +1,175 @@
++++
+title = "Authenticating Kubeflow to GCP"
+description = "Troubleshooting guide for authentication and authorization to GCP"
+weight = 4
++++
+
+
+# Authenticating gcloud
+
+[The `gcloud` tool](https://cloud.google.com/sdk/gcloud/) is used to interact with Google Cloud Platform (GCP) over the command line. 
+It can be used to [set up Google Kubernetes Engine (GKE) clusters](https://cloud.google.com/sdk/gcloud/reference/container/clusters/create), 
```

Comment:
Reduce the number of occurrences of passive voice on the page. (Passive voice results in longer sentences than active voice, and passive voice can also confuse the reader because the sentence doesn't make it clear who should perform the action.) 

I think the passive voice is fine in the first sentence, because we want to emphasize the `gcloud` tool. But in the second sentence, change this:

"It can be used to ..."

to this:

"You can use the `gcloud` command to ..."

(I've also swapped out the pronoun "it" to be more specific, for clarity.